internal/pkg/constants/constants.go

```diff
@@ -109,3 +111,8 @@ const (
 	// nolint: golint
 	SYSLOG_ACTION_READ_ALL = 3
 )
+
+// Containerd
+const (
+	ContainerdAddress = defaults.DefaultAddress
```

Comment:
Not that I'm opposed to it, but just curious what the motivation behind this was?

Reply:
The main intention was around being able to have a single place to update the containerd listen address. The defaults.DefaultAddress is used in a number of packages ( probably highlighting a need for a containerd abstraction package to consolidate things ), so by converting this into a constant it allows us some opportunities in the future for handling this.

Reply:
And more specifically, this is when I was trying to set containerd to listen on a tcp socket which didn't seem to work out the greatest.